### ISSUE TITLE
ArcPro 2.7 failing to execute sdf.spatial.to_featureclass()

### ISSUE THREAD

**Issue by Mungakha** [Esri/arcgis-python-api#890]
**Describe the bug**
Have been using ArcPro 2.6 to download feature layer from ArcGIS Online, write the resulting spatially enabled dataframe into a geodatabase on local disc to resource multiple business processes. I updated  to ArcPro 2.7 and the process now returns the following error ``` ValueError: invalid JSON data```. I reinstated 2.6.2 and now it works ok

**To Reproduce**
```
from arcgis.gis import GIS
import arcgis
from arcgis import features
from arcgis.geoanalytics import manage_data
from arcgis.features.manage_data import overlay_layers
from arcgis.features import GeoAccessor, GeoSeriesAccessor, FeatureLayer
import arcpy
import sys, os
import pandas as pd
import datetime as dt
from arcgis.features import FeatureLayerCollection
import http.client
import mimetypes
import json
import requests
from pandas import json_normalize
from arcgis import geometry 
from copy import deepcopy
from arcgis.features.manage_data import dissolve_boundaries


gis = GIS("url.maps.arcgis.com", "UserName", "Password)

item=gis.content.get('Feature_Layer Portal ID')
l=item.layers[0] 
df=l.query().sdf

#Create Temporary Geodatabase

if arcpy.Exists(r"C:\Centroid"):
   arcpy.Delete_management(r"C:\Centroid")
arcpy.CreateFolder_management(r"C:", "Centroid")

if arcpy.Exists(r"C:\Centroid"):
   arcpy.Delete_management(r"C:\Centroid\Centroid")
arcpy.CreateFolder_management(r"C:\Centroid", "Centroid")

arcpy.CreateFolder_management(r"C:","Centroid")
if arcpy.Exists(r"C:\Centroid\Centroid\KCentroids.gdb"):
   arcpy.Delete_management(r"C:\Centroid\Centroid\KCentroids.gdb")

arcpy.CreateFileGDB_management(r"C:\Centroid\Centroid","KCentroids")

#Write spatially enabled dataframe to disk

df.spatial.to_featureclass('C:\Centroid\Centroid\KCentroids.gdb\jana')
```
error:
```
ValueError: invalid JSON data
```
**Screenshots**

**Expected behavior**

**Platform (please complete the following information):**
 - Python API Version [e.g. `1.6.2`] (you can get this by typing `print(arcgis.__version__)`

**Additional context**
Add any other context about the problem here, attachments etc.

**Comment by priyankatuteja** [comment]
I tried to reproduce this issue. 
Screenshot below shows the same code run using ArcGIS Pro's Python command prompt (Version 2.6):
![image](https://user-images.githubusercontent.com/39754605/104728633-30f6fc80-575d-11eb-86ef-9453d2983a1e.png)

The same code is run using ArcGIS Pro (2.6) embedded notebook: 
![image](https://user-images.githubusercontent.com/39754605/104729248-28eb8c80-575e-11eb-8155-9a79d0382507.png)

![image](https://user-images.githubusercontent.com/39754605/104729292-37d23f00-575e-11eb-859e-a55162acb48c.png)

As mentioned, then I uninstalled Pro 2.6
Screenshot below shows version of Pro uninstalled:
![image](https://user-images.githubusercontent.com/39754605/104729574-a2837a80-575e-11eb-90a9-d41230af9d79.png)

Reinstalled Pro 2.7:

![image](https://user-images.githubusercontent.com/39754605/104735791-0c545200-5768-11eb-9998-5d09617cdbae.png)

Notebook server launched using Python  command prompt (Version 2.7):
![image](https://user-images.githubusercontent.com/39754605/104742771-dd8ea980-5770-11eb-9bcb-4aaab1ef9a12.png)

The same code is run using ArcGIS Pro (2.7) embedded notebook: 
![image](https://user-images.githubusercontent.com/39754605/104742393-60fbcb00-5770-11eb-9f27-76c24aac8f89.png)

![image](https://user-images.githubusercontent.com/39754605/104742449-73760480-5770-11eb-86ba-48ac969e6b01.png)

**Comment by priyankatuteja** [comment]
@Mungakha Can you try the code mentioned below and see if you still get some error:

```
from arcgis.gis import GIS
import arcgis
from arcgis import features
from arcgis.geoanalytics import manage_data
from arcgis.features.manage_data import overlay_layers
from arcgis.features import GeoAccessor, GeoSeriesAccessor, FeatureLayer
import arcpy
import sys, os
import pandas as pd
import datetime as dt
from arcgis.features import FeatureLayerCollection
import http.client
import mimetypes
import json
import requests
from pandas import json_normalize
from arcgis import geometry 
from copy import deepcopy
from arcgis.features.manage_data import dissolve_boundaries
gis = GIS("https://geosaurus.maps.arcgis.com", "arcgis_python", "P@ssword123")

item=gis.content.get('e1979a2137ca427690b2240262a84a5b')
item
l=item.layers[0] 
df=l.query().sdf
if arcpy.Exists(r"C:\Centroid"):
   arcpy.Delete_management(r"C:\Centroid")
arcpy.CreateFolder_management(r"C:", "Centroid")

if arcpy.Exists(r"C:\Centroid"):
   arcpy.Delete_management(r"C:\Centroid\Centroid")
arcpy.CreateFolder_management(r"C:\Centroid", "Centroid")

arcpy.CreateFolder_management(r"C:","Centroid")
if arcpy.Exists(r"C:\Centroid\Centroid\KCentroids.gdb"):
   arcpy.Delete_management(r"C:\Centroid\Centroid\KCentroids.gdb")

arcpy.CreateFileGDB_management(r"C:\Centroid\Centroid","KCentroids")

#Write spatially enabled dataframe to disk

df.spatial.to_featureclass('C:\Centroid\Centroid\KCentroids.gdb\jana')
```

If so, please open close any already open Python command prompts and close ArcGIS Pro as well Then uninstall and reinstall pro 2.7. After that try running the code mentioned above.

**Comment by Mungakha** [comment]
Trying with a ArcPro 2.7 downloaded from MyEsri on 21/12/2020 and installed on two different computers; my pc and a Virtual Machine in Azure, the error replicated itself

![image](https://user-images.githubusercontent.com/52562426/105119947-43ca5200-5b0c-11eb-87f4-ce8d1ce75e8b.png)

**Comment by Mungakha** [comment]
This was resolved by patch [2.7.1](https://community.esri.com/t5/arcgis-pro-documents/arcgis-pro-2-7-patch-1-is-available/ta-p/1022652)
